### ISSUE TITLE
My StopSmoking Coach does not launch as of 0.73.0 

### ISSUE THREAD

**Issue by Lesking72** [DS-Homebrew/nds-bootstrap#1635]
**Your console:** New 3DS

**Launch Method:** SD Card

**Version used:** Release 1.0.0

**Game tested/used/applicable:** My StopSmoking Coach (NTR-CUSE-USA) SHA1: 59FBFBD454C3B815ABF7112041F8CAFB83FFCA93

**Save file:** [My Stop Smoking Coach.zip](https://github.com/DS-Homebrew/nds-bootstrap/files/12707714/My.Stop.Smoking.Coach.zip)

**Expected Behavior:** Game launches

**Actual Behavior:** nds-bootstrap hangs on a white screen

### Steps to reproduce
1. Attempt to launch game with nds-bootstrap 0.73.0 or later

### nds-bootstrap.ini
````
[NDS-BOOTSTRAP]
DEBUG = 1
LOGGING = 1
B4DS_MODE = 0
ROMREAD_LED = 0
DMA_ROMREAD_LED = -1
PRECISE_VOLUME_CONTROL = 0
SDNAND = 0
MACRO_MODE = 0
SLEEP_MODE = 1
SOUND_FREQ = 0
CONSOLE_MODEL = 2
HOTKEY = 284
USE_ROM_REGION = 1
NDS_PATH = sd:/Roms/nds/My Stop Smoking Coach.nds
SAV_PATH = sd:/Roms/nds/saves/My Stop Smoking Coach.sav
RAM_DRIVE_PATH = sd:/null.img
GUI_LANGUAGE = en
LANGUAGE = 1
REGION = 1
DSI_MODE = 1
BOOST_CPU = 0
BOOST_VRAM = 0
CARD_READ_DMA = 1
ASYNC_CARD_READ = 0
DONOR_SDK_VER = 0
PATCH_MPU_REGION = 0
PATCH_MPU_SIZE = 0
FORCE_SLEEP_PATCH = 0
````

### Logs
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/12707720/NDSBTSRP.LOG)

### Other notes
Using a fresh installation of TWiLight Menu 26.1.0 from Universal Updater. Version seems to be irrelevant as simply downgrading nds-bootstrap to 0.72.1 makes the game start working. Also tested on another console where TWiLight Menu was updated from 25.8.1, where the game was previously working.

### Will you be available for future issues and/or replies?
Yes

**Comment by Lesking72** [comment]
Not fixed by 9761a4
[NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/12709186/NDSBTSRP.LOG)

**Comment by RocketRobz** [comment]
> Not fixed by `97691a4` [NDSBTSRP.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/12709186/NDSBTSRP.LOG)
> 
> Edit: Works when run in DSi mode on `97691a4`. Tested this on 1.0.0 and it didn't work. [NDSBTSRP_2.LOG](https://github.com/DS-Homebrew/nds-bootstrap/files/12709220/NDSBTSRP_2.LOG)

Strange that it's not working in DS mode on your side, as it works for me with both DS and DSi modes.
